docs/howto/create-a-release.rst

```diff
@@ -102,6 +102,115 @@ Upload build to PyPI
 
 Check PyPI for your upload.
 
+Upload to Conda
+===============
+
+NOTE: the GitHub and PYPI releases are prerequisites for releasing to
+Conda Forge!
+
+The Conda Forge release process is mostly automated by bots. These bots
+are set up during the initial project bootstrap, which has been done for
+all the projects that are part of Cylc 8.
+
+As there are inter-dependencies amongst the Cylc 8 parts, you should
+know the dependency tree, and start by the leaf nodes (i.e. a module
+A may have multiple dependants, but no dependency to other modules).
+
+For example, Cylc UI Server depends on Cylc Flow. So unless you are
+releasing only Cylc UI Server, you should release Cylc Flow first.
+
```

Comment:
Suggest that you add indication of current structure of dependencies:
![cylc-conda-deps png](https://user-images.githubusercontent.com/26465611/79469340-f4447c00-7ff7-11ea-888e-bd13b906ff2d.png)
(I have the svg of that if you want)
Also suggest that you tell people where to find the info they need -Am I correct in thinking that this is `github.com/conda-forge/${REPO}/blob/master/recipe/meta.yaml` for each of the repos shown below?

Reply:
>Suggest that you add indication of current structure of dependencies:

Maybe that could be a follow-up change? And I think this should be for both PYPI and Conda, as I think they would share the order/dependency relationship? What do you think?

>Am I correct in thinking that this is github.com/conda-forge/${REPO}/blob/master/recipe/meta.yaml for each of the repos shown below?

Yup! The link for each repository we have is listed below this text. Should I remove that, or modify it perhaps? Below the list, it points that the developer releasing it should look for `recipe/meta.yaml` (assuming s/he cloned the repo).

Reply:
Links are fine - I just need to read more carefully.
Perhaps:
```rst
- https://github.com/conda-forge/metomi-isodatetime-feedstock (`mainly the recipe/meta.yaml file <https://github.com/conda-forge/${REPO}/blob/master/recipe/meta.yaml>`_)
```

Reply:
I'll leave it as-is as I couldn't find an easy way to add the links in every list item :smile: but hopefully the paragraph below with the location of `meta.yaml` should help for now.

Reply:
BTW, one difference with conda is that it has two extra packages not present in PYPI, `cylc-ui` (JS) and `cylc` (metapackage with links to each dependency in the Cylc 8 release). If you share the SVG later or add it in a PR I can try to add these two too.